apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -242,6 +255,7 @@ const DanceAiModal: React.FunctionComponent = () => {
       JSON.stringify({
         inputs,
         ...generatedEffects.current.goodEffect,
+        minMaxAssociations: minMaxAssociations.current,
```

Comment:
Is it crucial that we store these?  Or can we regenerate them when reopening the modal?  We want to keep the stored properties as minimal as possible.  Given the new idea to regenerate the "bad" values when reopening the modal, hoping we might be able to regenerate these, too.

Reply:
Not crucial :) I updated to regenerate the scale on each render where we have "good" and "bad" effects selected, and not to serialize it. There's definitely a better React Hooks way to do this, but I'm struggling with it and I think I'd need to do some refactoring to get it to work (and don't want to generate too many merge conflicts). Happy to pair on it if you'd like though!